### ISSUE TITLE
Error on apt-get update

### ISSUE THREAD

**Issue by antis81** [qTox/qTox#1897]
After installing qTox from the nightly .deb package, I regularly get this error message:

``` bash
W: Size of file /var/lib/apt/lists/partial/repo.tox.im_dists_nightly_Release.gpg is not what the server reported 473 168
W: Fehlschlag beim Holen von http://de.archive.ubuntu.com/ubuntu/dists/vivid-updates/universe/binary-i386/Packages  Hash-Summe stimmt nicht überein
```

The German message says: "hash sum doesn't match".

Of course I don't want to remove the key and reinstall qTox every time this happens. Can you fix it on the server side without forcing users to reinstall?

**Comment by tux3** [comment]
Unfortunately I'm not the maintainer of the Linux packages, they've been broken for quite some time. I'm considering making my own packages, but I have little packaging experience and little time.

**Comment by antis81** [comment]
Hm ... something must have changed. I did not remove neither qTox nor the key and today I got a new update - everything looks fine.

Anyway, I'm also not an expert in packaging. Just wanted to let you know, so hopefully someone is able to fix it.

I really like the grade of integration and the idea of the .deb installer setting up the PPA and everything. Combined with Aptik, this survives a full system upgrade with the rolling release - otherwise PPA's will be deactivated.
